lib/common/common/src/defaults.rs

```diff
@@ -0,0 +1,4 @@
+use std::time::Duration;
+
+/// Default timeout for consensus meta operations.
+pub const CONSENSUS_DEFAULT_META_OP_WAIT: Duration = Duration::from_secs(10);
```

Comment:
```suggestion
pub const CONSENSUS_META_OP_WAIT: Duration = Duration::from_secs(10);
```

---

Why not simply `CONSENSUS_META_OP_WAIT`?

And the use it as:
```rust
use common::defaults;

fn example() {
    println!("{}", defaults::CONSENSUS_META_OP_WAIT);
}
```

Reply:
Good suggestion. Thanks!